client/src/components/FormsUI/Select/index.tsx

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import React from 'react';
 import { TextField, MenuItem } from '@material-ui/core';
 import { useField, useFormikContext } from 'formik';
 
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
Even though nothing much was changed by you here, I think this is a good opportunity to learn how to handle this case. The `eslint-disable` lines as well as our use of `any` as a type here go against why we use ESLint & Typescript in a project and we shouldn't be circumventing the problems they raise unless it's **absolutely a must**. 

The main concern here is that we want to forward some of the props (i.e. `...otherProps`) but we aren't sure what type to use for it. All material ui components will have a type for the props they accept (i.e. you can import `TextFieldProps` for `TextField` components prop type). You can use these types with the [typescript intersection operator (&)](https://www.typescriptlang.org/docs/handbook/2/objects.html#intersection-types) to add any custom props necessary. I created a small example in codesandbox [here](https://codesandbox.io/s/still-sun-2hcu8?file=/src/App.tsx). Let's please have a look and hopefully we can make the proper adjustments to ensure we don't need to use `eslint-disable`s and/or `any` as a type.